benchmarker.py

```diff
@@ -102,7 +118,7 @@ def save_to_html(results):
 	heading = "<th></th>" + "".join([ '<th style="font-style:bold;text-align:center;padding:5px 20px;">' + next + "</th>" for next in items])
 	html += "<tr>" + heading + "</tr>"
 	for i in range(len(languages)):
-		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + str(round(next, 3)) + " s" + "</td>" for next in results[i]])
+		row = "<td>" + languages[i] +  "</td>" + "".join(['<td style="padding:5px 20px;">' + "{:.3f}".format(next, 3) + " s" + "</td>" for next in results[i]])
```

Comment:
yes, you're right. No need to give precision inside `format` as it is `{:.3f}` already.